mangaki/mangaki/templates/mangaki/reco_list_dpp.html

```diff
@@ -0,0 +1,62 @@
+{% extends "base.html" %}
+
+{% block body %}
+{% if reco_list|length > 0 %}
+<div class"row">
+<div class=" col-md-12 col-sm-12">
+    <div class="well">
+        <p>
+            Voici quatre recommandations qui devraient vous
+            plaire.<br />
+            Notez-les puis rechargez la page pour en faire apparaître
+            d'autres.
+
+    Inclure la liste des "À voir" dans les recommandations : <input type="checkbox" name="reco_willsee_ok" {% if user.profile.reco_willsee_ok %} checked{% endif %}/>
+        </p>
+    </div>
+</div>
+<div class="row">
+        <div class="col-md-12 col-sm-12" align="center">
+            Légende :
+             <span class="label label-favorite"><span class="glyphicon glyphicon-star"></span> Mettre en favori</span>
+            <span class="label label-success"><span class="glyphicon glyphicon-thumbs-up"></span> J'aime</span>
+            <span class="label label-warning"><span class="glyphicon glyphicon-adjust"></span> Neutre</span>
+            <span class="label label-danger"><span class="glyphicon glyphicon-thumbs-down"></span> Je n'aime pas</span>
+            <span class="label label-primary"><span class="glyphicon glyphicon-eye-open"></span> Je veux lire</span>
+            <span class="label label-info"><span class="glyphicon glyphicon-eye-close"></span> Je ne veux pas lire</span>
+        </div>
+</div>
+<br />
+
+    <ul class="nav nav-tabs">
+        <li role="presentation"{% if category == 'all' %} class="active"{% endif %}><a href="/reco_dpp/">Tout</a></li>
+        <li role="presentation"{% if category == 'anime' %} class="active"{% endif %}><a href="/reco_dpp/?category=anime"><em>Anime</em> seulement</a></li>
+        <li role="presentation"{% if category == 'manga' %} class="active"{% endif %}><a href="/reco_dpp/?category=manga">Manga seulement</a></li>
+    </ul>
+    <br />
+<div class="alert alert-warning" role="alert" id="sorry" style="display: none"></div>
+{% include "works.html" with works=reco_list only %}
+<div class="row margin">
+    <div class="col-xs-12">
+        <a class="btn btn-primary btn-lg" href="/reco/?category={{ category }}&editor={{ editor }}" style="display: block; margin: auto">Pose-moi d'autres questions&nbsp;! <br class="visible-xs">(Après avoir noté celles-ci.)</a>
+    </div>
+</div>
+    <script>
+    sort_mode = '';
+    $.getJSON('/data/reco_dpp/{{ category }}/{{ editor }}.json', function(data) {
+        data.forEach(function(work, i) {
+            displayWork(i + 1, work);
+        });
+    });
+    $("[name='reco_willsee_ok']").bootstrapSwitch();
+    $("[name='reco_willsee_ok']").on('switchChange.bootstrapSwitch', function(event, state) {
+       $.post('/reco_willsee/', {reco_willsee_ok: state});
+       setTimeout(function(){
+          location.reload();
+       }, 500);
+    });
+    </script>
+{% else %}
+    <p class="well">Pour obtenir une recommandation, il faut au moins avoir <a href="/anime/">noté un <em>anime</em></a> !</p>
```

Comment:
Faux pour DPP ; on veut afficher la DPP principalement aux gens qui n'ont rien noté encore !